.travis.yml

```diff
@@ -28,3 +27,4 @@ install:
 
 script:
   - vendor/bin/phpspec run
+  - vendor/bin/phpunit tests
```

Comment:
Yes, it's why you need a `phpunit.xml.dist` file (and the `.dist` allow custom configuration not versioned if necessary).